fedimintd/src/fedimintd.rs

```diff
@@ -425,9 +404,18 @@ impl Fedimintd {
         std::process::exit(-1);
     }
 
+    #[allow(unused, clippy::unused_self)]
```

Comment:
This is used only for `fedimintd dev ...` commmands and needs an extra fix, so left it broken for now.